### ISSUE TITLE
Unused Variables

### ISSUE THREAD

**Issue by Fyb3roptik** [launchdarkly/roku-client-sdk#32]
Getting quite a few unused variable warnings

```
BRIGHTSCRIPT: WARNING: unused variable 'launchdarklylocalcounter' in anonymous function in pkg:/source/LaunchDarkly.brs(1024)
BRIGHTSCRIPT: WARNING: unused variable 'launchdarklyparamcipherkey' in function 'launchdarklyplaintextreader' in pkg:/source/LaunchDarkly.brs(1672)
BRIGHTSCRIPT: WARNING: unused variable 'launchdarklyparamauthkey' in function 'launchdarklyplaintextreader' in pkg:/source/LaunchDarkly.brs(1672)
BRIGHTSCRIPT: WARNING: unused variable 'launchdarklyparammessage' in anonymous function in pkg:/source/LaunchDarkly.brs(3138)
BRIGHTSCRIPT: WARNING: unused variable 'launchdarklylocalcipherkeybuffer' in anonymous function in pkg:/source/LaunchDarkly.brs(3183)
BRIGHTSCRIPT: WARNING: unused variable 'launchdarklylocalauthkeybuffer' in anonymous function in pkg:/source/LaunchDarkly.brs(3183)
BRIGHTSCRIPT: WARNING: unused variable 'launchdarklylocalhostname' in anonymous function in pkg:/source/LaunchDarkly.brs(3183)
```

**Comment by keelerm84** [comment]
Thank you for raising this concern. I have a PR (https://github.com/launchdarkly/roku-client-sdk/pull/33) open to address this. I will let you know once we have cut a release for this.

**Comment by keelerm84** [comment]
Fixed in v2.1.2

Thank you again for reporting this. Have a great week!
